assets/js/components/dashboard/Version2/DashboardAllTrafficWidgetV2.js

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
@kostyalmm all these components must be located in the `assets/js/modules/analytics/components/dashboard/Version2/` folder.